### PR TITLE
[MIG][14.0] portal_odoo_debranding

### PR DESCRIPTION
**Pull request by ivantodorovich**
Standard migration

### CODE REVIEW

**Review by hparfr — APPROVED** [review]

**Comment by francesco-ooops** [comment]
any chance to merge this?

**Comment by meigallodixital** [comment]
Will merge soon this?

**Comment by ivantodorovich** [comment]
It needs one more review (2 required)

Feel free to review it yourself ☺️ anyone can do it.

If you don't have technical knowledge, you can do a functional test on runbot. These tests are often more valuable than code reviews

**Comment by pedrobaeza** [comment]
https://odoo-community.org/page/review

**Review by francesco-ooops — APPROVED** [review]
LGTM

File: portal_odoo_debranding/i18n/portal_odoo_debranding.pot [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 12.0\n"
```
Comment (Shide) [inline portal_odoo_debranding/i18n/portal_odoo_debranding.pot:6]: Change 12 to 14

File: portal_odoo_debranding/README.rst [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-LGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+    :alt: License: LGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fserver--brand-lightgray.png?logo=github
+    :target: https://github.com/OCA/server-brand/tree/12.0/portal_odoo_debranding
```
Comment (Shide) [inline portal_odoo_debranding/README.rst:17]: Change 12 to 14

File: portal_odoo_debranding/README.rst [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    :alt: OCA/server-brand
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/server-brand-12-0/server-brand-12-0-portal_odoo_debranding
```
Comment (Shide) [inline portal_odoo_debranding/README.rst:20]: Change 12 to 14

File: portal_odoo_debranding/README.rst [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/252/12.0
```
Comment (Shide) [inline portal_odoo_debranding/README.rst:23]: Change 12 to 14

File: portal_odoo_debranding/README.rst [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+.. contents::
+   :local:
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/server-brand/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed
+`feedback <https://github.com/OCA/server-brand/issues/new?body=module:%20portal_odoo_debranding%0Aversion:%2012.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (Shide) [inline portal_odoo_debranding/README.rst:41]: Change 12 to 14

File: portal_odoo_debranding/README.rst [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+
+.. |maintainer-eLBati| image:: https://github.com/eLBati.png?size=40px
+    :target: https://github.com/eLBati
+    :alt: eLBati
+
+Current `maintainer <https://odoo-community.org/page/maintainer-role>`__:
+
+|maintainer-eLBati| 
+
+This module is part of the `OCA/server-brand <https://github.com/OCA/server-brand/tree/12.0/portal_odoo_debranding>`_ project on GitHub.
```
Comment (Shide) [inline portal_odoo_debranding/README.rst:81]: Change 12 to 14

File: portal_odoo_debranding/static/description/index.html [hunk 7]
```diff
@@ -0,0 +1,424 @@
...
+</ul>
+</li>
+</ul>
+</div>
+<div class="section" id="bug-tracker">
+<h1><a class="toc-backref" href="#id1">Bug Tracker</a></h1>
+<p>Bugs are tracked on <a class="reference external" href="https://github.com/OCA/server-brand/issues">GitHub Issues</a>.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed
+<a class="reference external" href="https://github.com/OCA/server-brand/issues/new?body=module:%20portal_odoo_debranding%0Aversion:%2012.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**">feedback</a>.</p>
```
Comment (Shide) [inline portal_odoo_debranding/static/description/index.html:389]: Change 12 to 14

File: portal_odoo_debranding/static/description/index.html [hunk 8]
```diff
@@ -0,0 +1,424 @@
...
+<div class="section" id="maintainers">
+<h2><a class="toc-backref" href="#id5">Maintainers</a></h2>
+<p>This module is maintained by the OCA.</p>
+<a class="reference external image-reference" href="https://odoo-community.org"><img alt="Odoo Community Association" src="https://odoo-community.org/logo.png" /></a>
+<p>OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.</p>
+<p>Current <a class="reference external" href="https://odoo-community.org/page/maintainer-role">maintainer</a>:</p>
+<p><a class="reference external" href="https://github.com/eLBati"><img alt="eLBati" src="https://github.com/eLBati.png?size=40px" /></a></p>
+<p>This module is part of the <a class="reference external" href="https://github.com/OCA/server-brand/tree/12.0/portal_odoo_debranding">OCA/server-brand</a> project on GitHub.</p>
```
Comment (Shide) [inline portal_odoo_debranding/static/description/index.html:418]: Change 12 to 14

File: portal_odoo_debranding/__manifest__.py [hunk 9]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2020 Lorenzo Battistini @ TAKOBI
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+{
+    "name": "Remove Odoo Branding from Website",
+    "version": "14.0.1.0.0",
+    "development_status": "Beta",
```
Comment (Shide) [inline portal_odoo_debranding/__manifest__.py:6]: You can remove development_status beta if you want (not blocking this)

**Review by Shide — CHANGES_REQUESTED** [review]
Wrong type of review.

**Review by Shide — APPROVED** [review]
Seems good to me

**Comment by yajo** [comment]
/ocabot merge nobump

Let's try after merging #46

**Pull request merged**